app/src/main/java/pl/futurecollars/invoicing/db/FileDatabase.java

```diff
@@ -4,20 +4,21 @@
 import java.util.List;
 import java.util.Optional;
 import lombok.RequiredArgsConstructor;
-import org.springframework.context.annotation.Primary;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import pl.futurecollars.invoicing.configuration.Config;
 import pl.futurecollars.invoicing.model.Invoice;
 import pl.futurecollars.invoicing.service.FileService;
 
-@Primary
+@Repository
 @RequiredArgsConstructor
 
 public class FileDatabase implements Database {
 
   private final FileService fileService;
 
-  private final String invoiceDbPath;
-  private final String idDbPath;
-
+  @Autowired
```

Comment:
jak masz @RequiredArgsConstructor to nie potrzebujesz @Autowired przy zmiennych private final... w teorii :)